website/source/guides/security/acl.html.markdown

```diff
@@ -419,6 +419,8 @@ Error bootstrapping: Unexpected response code: 500 (Invalid bootstrap reset inde
 
 This is because the reset file is in place, but with the incorrect index. The reset file can be deleted, but Nomad will not reset the bootstrap until the index is corrected.
 
+Resetting ACL Bootstrap does not automatically invalidate previous ACL tokens: Previous bootstrap token remains a valid, and existing tools that utilize it remain functional. If the token is unused, or if a management token is suspected of being vulnerable, then we should invalidate it, update any existing system with new tokens, and audit all existing tokens.
```

Comment:
Adding a callout to really drive the point home; small wording tweaks. 
```suggestion
~> **Note**: Resetting ACL Bootstrap does not automatically invalidate previous ACL tokens.
Resetting the ACL Bootstrap does not automatically invalidate any previous ACL tokens. All previous bootstrap and management token remains valid, and existing tools that utilize it will remain functional. If the token is unused or if a management token is suspected of being vulnerable, then you should invalidate it, update any existing system with new tokens, and audit all existing tokens.
```

Reply:
Oh, just realized we both posted fixes.

> then you should invalidate it

I thought "you" would be better as well, but the surrounding text uses "we".

Reply:
The new guide will move from "we" to "you", but it can be adjusted once there.  I'm not opposed to using "we" here, since it's used all over this version of the guide.